types.d.ts

```diff
@@ -0,0 +1,55 @@
+export type MontiEvent = any
+export type KadiraInfo = any
+
+export type ConnectOptions = {
+    enableErrorTracking?: boolean
+    endpoint?: string
+    hostname?: string
+    uploadSourceMaps?: boolean
+    recordIPAddress?: string
+    eventStackTrace?: boolean
+    disableNtp?: boolean
+    stalledTimeout?: number
+}
+
+export type TrackErrorOptions = {
+    type?: string
+    subType?: string
+    kadiraInfo?: KadiraInfo
+}
+
+export type TraceInfo = {
+    type: string
+    name: string
+}
+
+export type EventType = 'start' | 'end' | 'email' | 'db' | 'http' | 'fs' | 'compute' | 'custom'
+
+export namespace Tracer {
+    function addFilter(filterFunction: (eventType: EventType, data: Record<string, any>, info: TraceInfo) => any): void
+}
+
+export namespace Monti {
+    var tracer: typeof Tracer
+
+    function connect(appId: string, appSecret: string, options?: ConnectOptions): void;
+
+    function startContinuousProfiling(): void;
+    function enableErrorTracking(): void;
+    function disableErrorTracking(): void;
+
+    function trackError(error: Error, options?: TrackErrorOptions): void;
+
+    function ignoreErrorTracking(error: Error): void;
+
+    function startEvent(name: string, data?: Record<string, any>): MontiEvent | false;
+    function endEvent(event: MontiEvent | false, data?: any): void;
```

Comment:
The type for this `data` should probably match the type for the data in `startEvent`